app/models/user.rb

```diff
@@ -8,8 +8,8 @@ class User < ApplicationRecord
   has_many :comments, foreign_key: 'author_id', dependent: :destroy
 
   validates :name, presence: true
-  validates :photo, presence: true
-  validates :bio, presence: true
+  # validates :photo, presence: true
+  # validates :bio, presence: true
```

Comment:
- [ ] (OPTIONAL) Kindly remove all pieces of commented code to keep your work clean 👍